setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
@zoidbergwill I'm wasn't arguing against imports in `__init__.py`. I was saying that we shouldn't work around problems that we don't know we'll have because we might never have them.